pettingzoo/utils/wrappers/base.py

```diff
@@ -71,23 +78,23 @@ def action_spaces(self):
                 "The base environment does not have an action_spaces dict attribute. Use the environment's `action_space` method instead"
             ) from e
 
-    def observation_space(self, agent):
+    def observation_space(self, agent: str) -> gymnasium.spaces.Space:
         return self.env.observation_space(agent)
 
-    def action_space(self, agent):
+    def action_space(self, agent: str) -> gymnasium.spaces.Space:
         return self.env.action_space(agent)
 
     @property
-    def unwrapped(self):
+    def unwrapped(self) -> AECEnv:
         return self.env.unwrapped
 
     def close(self):
         self.env.close()
 
-    def render(self):
+    def render(self) -> None | np.ndarray | str | list:
```

Comment:
Should this be Any? What if the user wants to do a custom way of rendering which isnt str list or ndarray? Like a torch tensor or jax or something? Maybe @jjshoots or @RedTachyon have thoughts

Reply:
This would be a separate PR though, same with the action space change